Src/Workspaces/Core/Portable/Simplification/AbstractSimplificationService.cs

```diff
@@ -231,7 +231,7 @@ protected virtual SyntaxNode TransformReducedNode(SyntaxNode reducedNode, Syntax
             }, cancellationToken);
             }
 
-            Task.WaitAll(simplifyTasks, cancellationToken);
```

Comment:
No worries. I just wanted to make sure we were on the same page about what would be a deviation in behavior. Is the simple change leaving the behavior as-is, or removing the cancellationToken from the WhenAll? I'll do either one.